gradle.properties

```diff
@@ -1,7 +1,7 @@
 #Gradle properties
 quarkusPluginId=io.quarkus
-quarkusPluginVersion=2.12.1.Final
+quarkusPluginVersion=2.13.2.Final
```

Comment:
should this not be the same version as `quarkusPlatformVersion` - ie `2.13.3.FInal`?

Reply:
updated to match the bom